package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "datwd",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Please remove this version change. I'll bump it on main when I release.

Reply:
Done!